_posts/2019-09-06-kubernetes-and-hokusai.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: epic
+title: "Kubernetes and Hokusai"
+date: "2019-09-06"
+author: [matt-dole]
+categories: [artsy, beginners, engineering, hokusai, kubernetes, k8s]
+---
+
+When I joined Artsy Engineering a few months ago, I had roughly 0 knowledge of Kubernetes. I'd heard the term
+thrown around a few times, but had no idea how it worked or what it was good for.
+
+Artsy also has a super cool wrapper that we use to emulate a lot of Heroku CLI functionality for Kubernetes (and to
+do some other awesome stuff, read on for deets). It's fully open-source, and you should think about adopting it if
```

Comment:
Is `hokusai` a heroku-cli emulation? It seems like it's its own thing and is _similar to_ example x/y/z (would be good to dig up other examples more specifically related to Kubernetes orchestration and mention how it might relate to common workflows found within heroku cli).

Reply:
It's not specifically emulating `heroku`, but from discussions with @izakp, Hokusai was developed with the _ease_ of the `heroku` CLI in mind (and imo, he succeeded).

Reply:
good point - I think this is addressed more explicitly in the `What is Hokusai` section, and I removed this particular part from the intro